events/bancho.py

```diff
@@ -123,6 +122,13 @@ async def ping(p: Player, pr: BanchoPacketReader) -> None:
     # ping.. this handler shouldn't even exist lol
     p.ping_time = int(time.time())
 
+    # osu! seems to error when i send nothing back,
+    # so perhaps the official bancho implementation
+    # expects something like a stats update.. i'm
+    # just gonna ping it back, as i don't really
+    # want to something more expensive so often lol
+    p.enqueue(b'\x04\x00\x00\x00\x00\x00\x00')
```

Comment:
still quite unsure about this, i'll keep it in the back of my mind if anything strange starts happening lol